### PR TITLE
feat: add dark theme

### PR DESCRIPTION
**Pull request by WailanTirajoh**
1. add app-dark.scss to nuxt config
2. change import path from ~ to node_modules at app-dark.scss
3. create dark component (using function from main mazer dark.js)
4. add dark component to layout sidebar

## Todos

- [ ]

### CODE REVIEW

**Comment by fzn0x** [comment]
This is very awesome :) !

File: assets/scss/themes/dark/app-dark.scss [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
-    @import "~bootstrap/scss/close";
-    @import "~bootstrap/scss/toasts";
-    @import "~bootstrap/scss/modal";
-    @import "~bootstrap/scss/tooltip";
-    @import "~bootstrap/scss/popover";
-    @import "~bootstrap/scss/carousel";
-    @import "~bootstrap/scss/spinners";
-    @import "~bootstrap/scss/offcanvas";
-    @import "~bootstrap/scss/placeholders";
+//     // Bootstrap
```
Comment (fzn0x) [inline assets/scss/themes/dark/app-dark.scss:10]: Can you revert this change?

File: assets/scss/themes/dark/app-dark.scss [hunk 2]
```diff
@@ -1,52 +1,52 @@
...
+    @import "node_modules/bootstrap/scss/spinners";
+    @import "node_modules/bootstrap/scss/offcanvas";
+    @import "node_modules/bootstrap/scss/placeholders";
 
-    
-    
-    // Mazer CSS
+
+
+//     // Mazer CSS
```
Comment (fzn0x) [inline assets/scss/themes/dark/app-dark.scss:46]: Can you revert this change?

File: assets/scss/themes/dark/app-dark.scss [hunk 3]
```diff
@@ -1,52 +1,52 @@
...
     @import "./../../mazer";
-    
-    // Additional dark-mode CSS
+
+//     // Additional dark-mode CSS
```
Comment (fzn0x) [inline assets/scss/themes/dark/app-dark.scss:49]: Can you revert this change?

**Review by fzn0x — COMMENTED** [review]
Other than this, looks good!

File: assets/scss/themes/dark/app-dark.scss (hunk 3 above)
Comment (WailanTirajoh) [reply assets/scss/themes/dark/app-dark.scss]: sorry forgot to uncomment the comment while debug, ady push the comment, please review, thanks!

**Review by fzn0x — APPROVED** [review]
Looks good to me
